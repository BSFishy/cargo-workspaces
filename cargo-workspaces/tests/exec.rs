mod utils;
#[cfg(unix)]
use insta::assert_snapshot;

#[cfg(unix)]
#[test]
fn test_normal() {
    let (out, err) = utils::run("../fixtures/normal", &["ws", "exec", "cat", "Cargo.toml"]);
    assert_snapshot!(out);
    assert_snapshot!(err);
}
