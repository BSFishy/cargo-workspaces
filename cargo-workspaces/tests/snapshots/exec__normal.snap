---
source: tests/exec.rs
expression: out
---
[package]
name = "dep1"
version = "0.1.0"
authors = ["Pavan Kumar Sunkara <pavan.sss1991@gmail.com>"]
edition = "2018"

[dependencies]
[package]
name = "dep2"
version = "0.1.0"
authors = ["Pavan Kumar Sunkara <pavan.sss1991@gmail.com>"]
edition = "2018"

[dependencies]
pre_dep1 = { version = "0.1.0", path = "../dep1", package = "dep1" }
[package]
name = "top"
version = "0.1.0"
authors = ["Pavan Kumar Sunkara <pavan.sss1991@gmail.com>"]
edition = "2018"

[dependencies]
dep = { version = "0.1.0", path = "../dep1", package = "dep1" }
dep2 = { version = "0.1.0", path = "../dep2" }

